### DISCUSSION TITLE
Doesn't the current license require games made with OXR to share source code?

### DISCUSSION THREAD

**Discussion by gcmartinelli** [OpenXRay/xray-16#1569] [General]
Maybe I'm misinterpreting the license; but Stalker mod projects like G.A.M.M.A and True Stalker technically have to share their source code in order to use OpenXRay?

**Comment by Xottab-DUTY** [comment]
The licence is based on 2-clause BSD with added third clause that prohibits any commercial use. So, no, it doesn't require to share the project's source code.
But, technically, this licence was only made to protect GSC Game World property, and doesn't suit OpenXRay needs. So, in future, the licence may change to Apache or MIT/other permissive licence. We are investigating how we can do this.
